pkg/reconciler/testing/v1beta2/eventtype.go

```diff
@@ -60,12 +60,11 @@ func WithEventTypeType(t string) EventTypeOption {
 	}
 }
 
-func WithEventTypeBroker(broker string) EventTypeOption {
-	return func(et *v1beta2.EventType) {
-		et.Spec.Broker = broker
-	}
-}
-
+//	func WithEventTypeBroker(broker string) EventTypeOption {
+//		return func(et *v1beta2.EventType) {
+//			et.Spec.Broker = broker
+//		}
+//	}
```

Comment:
```suggestion
func WithEventTypeReference(ref *duckv1.KReference) EventTypeOption {
	return func(et *v1beta2.EventType) {
		et.Spec.Reference = ref
	}
}
```

Should we also add `WithEventTypeReference` to v1beta2 like we did for v1